### PR TITLE
assembler in c with instructions file, output file, Makefile and a li…

### PR DESCRIPTION
**Pull request by ChrisDragoi**
pls review

### CODE REVIEW

File: Assembler/description.txt
Comment (robert-gabriel-gaube) [inline Assembler/description.txt:1]: Change this to a README.md file

File: Assembler/description.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Description
+
+The assembler.c reads lines from instructions.txt and writes the binary inplementation in out.txt. 
+Addresses must be numbers as well as the immediate numbers. For MOV instruction, if u want to make move from
```
Comment (robert-gabriel-gaube) [inline Assembler/description.txt]: You, not u

**Review by robert-gabriel-gaube — CHANGES_REQUESTED** [review]

**Review by FermendjinPatricia — APPROVED** [review]
lgtm

**Review by robert-gabriel-gaube — APPROVED** [review]
LGTM

**Pull request merged**